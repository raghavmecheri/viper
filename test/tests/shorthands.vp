int a = 10;
int b = 20;
int c = 0;

/* 0 */
print(c);

/* 10 */
c += a;
print(c);

/* -10 */
c -= b;
print(c);

/* 190 */
c += a * b;
print(c);

/* -10 */
c -= a * b;
print(c);

/* 100 */
c *= c;
print(c);