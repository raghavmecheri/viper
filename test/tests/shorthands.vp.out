0
10
-10
190
-10
100
